src/libmongoc/doc/mongoc_handshake.rst

```diff
@@ -0,0 +1,80 @@
+:man_page: mongoc_handshake_data_append
+
+mongoc_handshake_data_append()
+=========================================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+   bool
+   mongoc_handshake_data_append (const char *driver_name,
+                                 const char *driver_version,
+                                 const char *platform);
+
+Appends the given strings to the handshake data for the underlying C Driver.
+
+Description
+-----------
+
+This function is intended for use by drivers which wrap the C Driver.
+Calling this function will store the given strings as handshake data about
+the system and driver by appending them to the handshake data for the
+underlying C Driver. These values, along with other handshake data collected
+during mongoc_init will be sent to the server as part of the initial
+connection handshake in the "client" document. This function must not be
+called more than once, or after a handshake has been initiated.
+
+The passed in strings are copied, and don't have to remain valid after the
+call to mongoc_handshake_data_append(). The driver may store truncated
+versions of the passed in strings.
+
+Note:
```

Comment:
Oh, good idea noting this! 

RST provides a `note` directive to indent and format. This may be applicable here.

```suggestion
.. note::
```